site/components/pages/Infinity/Cards/CardsSlider.tsx

```diff
@@ -45,19 +45,6 @@ const useWidth = (elementRef: RefObject<HTMLElement>) => {
             .getPropertyValue("border-right-width")
         );
 
-      width =
-        width -
-        parseInt(
-          window
-            .getComputedStyle(elementRef.current)
-            .getPropertyValue("padding-left")
-        ) -
-        parseInt(
-          window
-            .getComputedStyle(elementRef.current)
-            .getPropertyValue("padding-right")
-        );
-
```

Comment:
So this was the slider bug?
For the future: I would do this in a separate commit. 
This makes it much easier for reviewers to understand what your thought processes were and which code belongs to what exactly. 🙇

Reply:
Ok! that makes sense. Yea this was the slider bug. I will separate bugs and what not into their own PRs in the future.